saleor/graphql/product/mutations/products.py

```diff
@@ -891,7 +892,12 @@ def save(cls, info, instance, cleaned_input):
         if attributes:
             AttributeAssignmentMixin.save(instance, attributes)
             generate_and_set_variant_name(instance, cleaned_input.get("sku"))
-        info.context.plugins.product_updated(instance.product)
+
```

Comment:
It is only a detail but maybe soemthing more readable:
```
event_to_call = (
    info.context.plugins.product_variant_created if new_variant else 
    info.context.plugins.product_variant_updated
)
transaction.on_commit(lambda: event_to_call(instance))
```
@fowczarek  What do you think?

Reply:
In my opinion, the solution proposed by @korycins is more readable.